test/elements.js

```diff
@@ -49,7 +49,7 @@ test('svg', function (t) {
   </svg>`
   t.equal(result.tagName, 'svg', 'create svg tag')
   t.equal(result.childNodes[0].tagName, 'rect', 'created child rect tag')
-  t.equal(result.childNodes[1].getAttribute('xlink:href'), '#test', 'created child use tag with xlink:href')
+  t.equal(result.childNodes[2].getAttribute('xlink:href'), '#test', 'created child use tag with xlink:href')
```

Comment:
> As a practical matter, any HTML element can end up with white-space: preserve

Good point, didn't consider that. I wonder how common that is tho, and if that couldn't replaced by using a `<pre>` tag instead (unless we're talking about the most edgy of edge cases.

> Unless we start using other tag names to intelligently guess when space between elements should be removed

I think indeed that including these tags might indeed be the way to go